skaffold.yaml

```diff
@@ -31,23 +65,45 @@ build:
     context: src/contacts
   - image: gcr.io/bank-of-anthos/userservice
     context: src/userservice
+deploy:
+  kubectl:
+    manifests:
+    - dev-kubernetes-manifests/balance-reader.yaml
+    - dev-kubernetes-manifests/contacts.yaml
+    - dev-kubernetes-manifests/ledger-writer.yaml
+    - dev-kubernetes-manifests/transaction-history.yaml
+    - dev-kubernetes-manifests/userservice.yaml
+
+---
+
+apiVersion: skaffold/v2beta18
+kind: Config
+metadata:
+  name: frontend # module defining frontend service
+requires:
+  - configs: [backend]
+build:
+  artifacts:
+  - image: gcr.io/bank-of-anthos/frontend
+    context: src/frontend
+deploy:
+  kubectl:
+    manifests:
+    - dev-kubernetes-manifests/frontend.yaml
+
+---
+
+apiVersion: skaffold/v2beta18
+kind: Config
+metadata:
+  name: loadgenerator # module defining a load generator service
+requires:
+  - configs: [backend]
+build:
+  artifacts:
   - image: gcr.io/bank-of-anthos/loadgenerator
     context: src/loadgenerator
-  - image: gcr.io/bank-of-anthos/accounts-db
-    context: src/accounts-db
-  - image: gcr.io/bank-of-anthos/ledger-db
-    context: src/ledger-db
-  tagPolicy:
```

Comment:
It looks like in this modular version we have lost the settings for tagPolicy, local concurrency, and deploy statusCheckDeadlineSeconds

Reply:
the `tagPolicy` defaults to `gitCommit` so it doesn't need to be explicitly set in the configs. Same with `statusCheckDeadlineSeconds` which defaults to `600 seconds` in newer versions of skaffold (vs. the `120 seconds` it was previously).
Build concurrency also has a flag now called `build-concurrency` so the project default  can be left to build consecutively (less resource intensive default), which can be made to build in parallel by running with `--build-concurrency=0` flag.

WDYT?

Reply:
I like the idea of relying on the default values (for `tagPolicy` and `statusCheckDeadlineSeconds`).
It will result in a less clustered `skaffold.yaml`

Currently looking into why `name: local-code` was introduced...